docs/tutorial/first-app.md

```diff
@@ -104,6 +113,14 @@ RewriteCond %{REQUEST_FILENAME} !-f
 RewriteRule . index.php [L]
 ```
 
+nginx does not use `.htaccess` files, so you will need to add the following to your server configuration in the `location` block:
+
+```
+if (!-e $request_filename){
+    rewrite ^(.*)$ /index.php break;
```

Comment:
Sure, but before that in the tutorial it tells you to create an index.php:

https://github.com/jkochis/Slim-Website/blob/gh-pages/docs/tutorial/first-app.md#create-the-application

I'm glad to contribute in any way I can, but I don't want to alter the docs too much from the original. My goal was to simply add lines for nginx users. I feel adding that could be TMI. What do you think?